src/theme/QueryTable/QueryTable.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Table from '../table';
+import { LoadNewButton } from './LoadNewButton';
+import styles from '../table/stickyHeader/stickyHeader.css';
+
+// eslint-disable-next-line import/prefer-default-export
+export const QueryTable = ({
+  queryHook,
+  queryConfig,
+  button,
+  scrollToSelector,
+  ...props
+}) => {
+  const {
+    data: response,
+    error,
+    isLoading,
+    isFetching,
+    fetchNextPage,
+    hasNextPage,
+    hasUpdate,
+  } = queryHook(queryConfig);
+
+  const handleClick = () => {
+    // When the header is fixed at the top, the position is 50px
+    // Therefore the page should only scroll into the view if the header is not at the top
+    if (document.querySelector(`.${styles.header}`).getBoundingClientRect().top - window.scrollY <= 50) {
+      document.querySelector(scrollToSelector).scrollIntoView(true);
+    }
+    // Update the table with new data
+    // button.onClick();
```

Comment:
@reyraa The function for updating the table isn't available yet so I guess the `button.onClick` line should be removed since the other comment is there